packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -57,6 +57,88 @@ import 'theme_data.dart';
 /// ```
 /// {@end-tool}
 ///
+/// ## CheckboxListTile isn't exactly what I want
+///
+/// If the way CheckboxListTile pads and positions its elements isn't quite
+/// what you're looking for, you can create custom labeled checkbox widgets by
+/// combining [Checkbox] with other widgets, such as [RichText] and [Padding].
+///
+/// {@tool snippet --template=stateful_widget_scaffold}
+///
+/// Here is an example of a custom LabeledLinkedCheckbox widget, but you can
+/// easily make your own configurable widget.
+///
+/// ```dart code-imports
+/// import 'package:flutter/gestures.dart';
+/// ```
+/// ```dart preamble
+/// class LabeledLinkedCheckbox extends StatelessWidget {
```

Comment:
I didn't think twice when i made up this name, LinkedLabelCheckbox may be a better name.

Reply:
I'll change it, I think it's better if the readers are less confused by the naming :smile: